scripts/changelog-for-patch.mjs

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env node
+
+import path from "node:path";
+import minimist from "minimist";
+import semver from "semver";
+import {
+  changelogUnreleasedDirPath,
+  changelogUnreleasedDirs,
+  getEntries,
+  printEntries,
+  replaceVersions,
+} from "./utils/changelog.mjs";
+
+const { previousVersion, newVersion } = parseArgv();
+
+const entries = changelogUnreleasedDirs.flatMap((dir) => {
+  const dirPath = path.join(changelogUnreleasedDirPath, dir.name);
+  return getEntries(dirPath);
+});
+
+console.log(
+  replaceVersions(
+    printEntries(entries).join("\n\n"),
+    previousVersion,
+    newVersion,
+    /** isPatch */ true
+  )
+);
+
+function parseArgv() {
+  const argv = minimist(process.argv.slice(2));
+  const previousVersion = argv["prev-version"];
```

Comment:
Shouldn't previous version always `require("prettier").version`?

Reply:
I think that's correct in most cases. However, the release script actually get the previous version in a different way(ref: https://github.com/prettier/prettier/blob/acf9242dd142a042d889ccdac649c367cc76b910/scripts/release/release.js#L20-L24 ). So I think it is safer to be able to specify it as an argument.

Reply:
@fisker What do you think? I want to merge this for next patch version.